src/useForm.test.tsx

```diff
@@ -960,11 +960,23 @@ describe('useForm', () => {
     it('should remove error', () => {
       const { result } = renderHook(() => useForm<{ input: string }>());
       act(() => {
-        result.current.setError('input', 'test');
+        result.current.setError('input', 'test', 'message');
         result.current.clearError('input');
       });
       expect(result.current.errors).toEqual({});
     });
+
+    it('should remove nested error', () => {
+      const { result } = renderHook(() => useForm<{ input: string }>());
```

Comment:
Can you fix generics parameter?

```suggestion
      const { result } = renderHook(() =>
        useForm<{ input: { nested: string } }>(),
      );
```